app/controllers/chargeback_controller.rb

```diff
@@ -305,11 +305,16 @@ def cb_rates_delete
         end
       end
       process_cb_rates(rates, "destroy")  unless rates.empty?
-      add_flash(_("The selected %{records} were deleted") %
-        {:records => ui_lookup(:models => "ChargebackRate")}, :info, true) unless flash_errors?
-      cb_rates_list
-      @right_cell_text = _("%{typ} %{model}") % {:typ => x_node.split('-').last, :model => ui_lookup(:models => "ChargebackRate")}
-      replace_right_cell([:cb_rates])
+      if flash_errors? && @flash_array.count == 1
```

Comment:
@chessbyte Check added for when a single assigned rate is selected for deletion from the accordion list, UI needs to stay in the accordion list and not go to summary list in the right cell (displaying ALL rates).